package.json

```diff
@@ -83,7 +83,7 @@
 		"@wordpress/api-fetch": "7.0.1",
 		"@wordpress/autop": "4.0.1",
 		"@wordpress/blob": "4.0.1",
-		"@wordpress/block-directory": "5.0.6",
+		"@wordpress/block-directory": "5.0.7",
```

Comment:
Looks like this package depends on `@wordpress/editor`, so this change is legit.